### PR TITLE
General Questions/Feedback and a text update WRT ClusterTracker/matUtils

### PR DESCRIPTION
**Pull request by jmcbroome**
Hi Sidney- I played around some with your demo and a local development server version of Galago and have a handful of points of feedback/questions. I also have a small text edit proposed here, specifically with regards to the matUtils cluster method, clarifying what the method is and linking my recently published manuscript on it to reference for more information.

Additional feedback/questions:

- Have you considered using a timetree as your base for analysis? It might make a nicer visual paired with your time-based graph. Theo Sanderson’s Chronumental is reportedly pretty sweet for SARS-CoV-2 big trees.

- The data representation and assumption sections are nice additions. Have you considered metadata-informed downsampling to try to reduce the bias, since you’re often working with relatively limited subsets of the data anyways? Or would that feel like you’re obfuscating all available information too much?

- Adding to the text box serially based on multiple metadata types is a nice solution to users wanting N numbers of groups of samples to be selected simultaneously, but it’s a little unintuitive in how it updates immediately and the pane must be exited by clicking out when the user is satisfied. Simply withholding the update until the user clicks a confirmation button at the bottom of the panel, which applies the new parameters and closes the panel could be useful (especially if the user tries to click back and forth between different clustering methods and the site freezes up as it computes matUtils heuristic clusters, though it doesn’t seem to take any time in the current demo. It’s a little unclear what features are currently available). 

Let me know your thoughts and if there's anything I could assist with!

### CODE REVIEW

**Review by sidneymbell — CHANGES_REQUESTED** [review]

File: src/components/cladeSelection/clusteringMethodSelect.tsx [hunk 1]
```diff
@@ -163,10 +163,10 @@ function ClusteringOptions() {
                   size="small"
                 />
               }
-              label="Matutils"
+              label="Introduction Weight Heuristic"
             />
             <FormHelperText>
-              Matutils is a heuristic-based method. <br />
+              Introduction Weight is a simple heuristic-based clustering method, originally implemented in <a href="https://academic.oup.com/ve/article/8/1/veac048/6609172">Cluster-Tracker</a>. <br />
```
[the remark this answers is not in the document]
Comment (sidneymbell) [reply src/components/cladeSelection/clusteringMethodSelect.tsx:169]: (It's already got a heading that these are clustering methods :)

**Comment by sidneymbell** [comment]
cc @happyimadesignr

File: src/components/cladeSelection/clusteringMethodSelect.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (jmcbroome) [reply src/components/cladeSelection/clusteringMethodSelect.tsx:169]: Sure!

**Comment by jmcbroome** [comment]
RE Point 1, let me know once it's live and I'll check it out! I reference timetree specifically so that it can appear in line with your existing graph, though if its unrooted I suppose that doesn't apply. 

RE Point 2, it's possible to remove samples that are identical to other samples and collected from the same region at about the same time, at the most basic level, which is pretty safe. You could even replace these samples with a number of condensed or collapsed samples on your tree visual, potentially... definitely could use some more thought.

RE Point 3, I mean the panel that appears when "Filter and Suggest Clades" is clicked, where the text edit I proposed is and so on, if that's what you're referring to.

**Pull request merged**